### PR TITLE
Add committee picture

### PR DESCRIPTION
**Pull request by verwoerd**

### CODE REVIEW

**Comment by verwoerd** [comment]
image

File: content/contact.md [hunk 1]
```diff
@@ -12,16 +12,26 @@ The organisation can be contacted at [contact@2022.nwerc.eu](mailto:contact@2022
...
-* Maarten Weyns
-* Brian Witmer
+The CHipCie takes care of the programming contest within Delft and consist of the following people.
+![CHipCie](/chipcie.webp)
+
+## Top Row
+From left to right:
+* Nadyne Aretz, _Chairwoman_
+* Laura Pircalaboiu, _Treasurer_
+* Raoul Bruens, _Commisioner of Logistics & Activities_
```
Comment (mpsijm) [inline content/contact.md:22]: Commis**s**ioner is with double "s" :wink:

File: content/contact.md [hunk 2]
```diff
@@ -12,16 +12,26 @@ The organisation can be contacted at [contact@2022.nwerc.eu](mailto:contact@2022
...
+* Brian Witmer, _Commisioner of Recruitment & Promotion_
+* Maarten Sijm, _Commisioner of Jury Affairs_
+
+## Middle row
+From left to right:
+* Robert van Dijk, _Commisioner of Systems_
+* Elise Deen, _Secretary_
+* Maarten Weyns, _Commisioner of System Affairs_
```
Comment (mpsijm) [inline content/contact.md:30]: Is there a difference between "Systems" and "System Affairs"?

**Review by mpsijm — CHANGES_REQUESTED** [review]

**Comment by mpsijm** [comment]
The bottom row is not entirely aligned; Merel is nicely placed in the center, but Jaden's chips bag is off-center :thinking:

File: content/contact.md (hunk 2 above)
Comment (verwoerd) [reply content/contact.md:30]: yes, read the document on the drive.

File: content/contact.md [hunk 3]
```diff
@@ -19,19 +19,19 @@ The CHipCie takes care of the programming contest within Delft and consist of th
 From left to right:
 * Nadyne Aretz, _Chairwoman_
 * Laura Pircalaboiu, _Treasurer_
-* Raoul Bruens, _Commisioner of Logistics & Activities_
-* Brian Witmer, _Commisioner of Recruitment & Promotion_
-* Maarten Sijm, _Commisioner of Jury Affairs_
+* Raoul Bruens, _Commissioner  of Logistics & Activities_
```
Comment (mpsijm) [inline content/contact.md:22]: You introduced an extra space :innocent: Probably doesn't matter for the rendered webpage, but still :joy:

**Review by mpsijm — APPROVED** [review]

**Pull request merged**